Constants.js

```diff
@@ -1,4 +1,11 @@
 export default {
   ROOT_URL: 'https://peoplepower-node.herokuapp.com',
-  PDF_DELETE_DELAY: 20
+  BILLS_BASE_URL: 'https://utilityapi.com/api/v2/bills',
+  EBCE_REBATE_STRING: 'Credited to (Debited from) NEM Balance',
```

Comment:
I left the above two constants where they were because they were rather specific features of how the utilityAPI code functioned and it felt like it actually be MORE inconvenient to store them here than in the same file when debugging. 

Just an opinion really, up to you! I can add back in my new PR

Reply:
I suppose this is coming from a view where the constants file are like tweak-able aspects of the server. These are things we likely wouldn't tweak without editing the code too.